### PR TITLE
Issue #13213: Remove '//ok' comments from InputUnnecessaryParenthesesIfStatement2

### PR DESCRIPTION
**Pull request by CubeTures**
Issue #13213 
I will get the suppressions too.

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -201,32 +201,6 @@
             files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisValidateOnlyOverlappingFalse.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisValidateOnlyOverlappingTrue2.java"/>
-  <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:204]: Rest all lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
Please restrict your PR to a single commit. Squash and rebase as necessary

Look at the failing CI : 

```
[ERROR] Failures: 
[ERROR]   CommitValidationTest.testCommitMessageHasProperStructure:216 Commit 5a5c99b0620e0946ffcc964766c287489e818e32 message: "Update InputUnnecessaryParenthesesIfStatement2.java\n\nAccidentally clipped off brace added back" is invalid
Proper commit message should adhere to the following rules:
    1) Must match one of the following patterns:
        ^Issue #\d+: .*$
        ^Pull #\d+: .*$
        ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    2) It contains only one line of text
    3) Must not end with a period, space, or tab
    4) Commit message should be less than or equal to 200 characters

The rule broken was: 1

```
Checkstyle requires your commit message to adhere to certain rules. Please modify accordingly.
Make sure to test your build using `mvn clean verify` before pushing changes.

**Comment by CubeTures** [comment]
Squashed! Thank you for your help!

**Comment by romani** [comment]
CI is red, please fix

**Comment by CubeTures** [comment]
I'm failing the ci/circle: run-inspections, but it still says build success. I'm unsure how to fix this issue since all of the code seems right and it still builds fine. I found the error /home/circleci/project/src/main/java/com/puppycrawl/tools/checkstyle/AbstractAutomaticBean.java uses unchecked or unsafe operations. and I'm not sure of what else could be causing the problems.

File: config/checkstyle-input-suppressions.xml [hunk 2]
```diff
@@ -201,32 +201,6 @@
...
-            files="checks[\\/]coding[\\/]unnecessaryparentheses[\\/]InputUnnecessaryParenthesesIfStatement2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]unnecessaryparentheses[\\/]InputUnnecessaryParenthesesIfStatement2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]unnecessaryparentheses[\\/]InputUnnecessaryParenthesesIfStatement2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]unnecessaryparentheses[\\/]InputUnnecessaryParenthesesIfStatement2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]unnecessaryparentheses[\\/]InputUnnecessaryParenthesesIfStatement2.java"/>
-  <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:228]: Everything lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
> I'm failing the ci/circle: run-inspections, but it still says build success. I'm unsure how to fix this issue since all of the code seems right and it still builds fine. I found the error

The failing CI are known and unrelated to your changes. Please ignore

**Review by nrmancuso — APPROVED** [review]
Ok to merge as CI passes

**Pull request merged**